code/modules/clothing/chameleon.dm

```diff
@@ -161,7 +161,7 @@
 			I.sprite_sheets = P.sprite_sheets
 			qdel(P)
 
-		if(istype(I, /obj/item/clothing) && istype(initial(picked_item), /obj/item/clothing))
```

Comment:
Part of me wants to say "you need to make OD compatible with the wack inconsistencies with BYOND", but the rest of me says "why the hell is this in our codebase"

Reply:
![image](https://user-images.githubusercontent.com/5714543/153055909-9b6fc0b7-45c5-465f-ba5a-7aca70ed7103.png)